.github/workflows/cancel_duplicates.yml

```diff
@@ -1,25 +1,33 @@
 name: Cancel Duplicates
 on:
-  workflow_run:
-    workflows: ["CI"]
-    types: ["requested"]
+  # Checks for duplicate jobs on every push (merge) and every 10 minutes
+  push:
+  schedule:
+    - cron: "*/10 * * * *"
 
 jobs:
-  cancel-duplicate-workflow-runs:
-    name: "Cancel duplicate workflow runs"
-    runs-on: ubuntu-latest
+  cancel-duplicate-runs:
+    name: Cancel duplicate workflow runs
+    runs-on: ubuntu-20.04
+    continue-on-error: true
```

Comment:
It's better to not error. You can use output and if: conditional in the cancel action

```suggestion
```

Reply:
There was actually two steps after the error before I migrated this to directly referring your workflow (since I need to checkout the code). And I also wasn't sure whether ` >= 20` will work for workflow `if` conditions.

Thanks for the suggestion. I'll try it out!